Code/Editor/Plugins/EditorAssetImporter/AssetImporterPlugin.cpp

```diff
@@ -102,21 +99,53 @@ AZStd::unique_ptr<AZ::DynamicModuleHandle> AssetImporterPlugin::LoadSceneLibrary
 
 QMainWindow* AssetImporterPlugin::EditImportSettings(const AZStd::string& sourceFilePath)
 {
-    const QtViewPane* assetImporterPane = GetIEditor()->OpenView(m_toolName.c_str());
-
-    if(!assetImporterPane)
+    if (!m_assetImporterWindow)
     {
         return nullptr;
     }
 
```

Comment:
This check may not be necessary anymore

Reply:
It's not really necessary, but I thought it would be quicker to return if `m_assetImporterWindow` is a nullptr so we wouldn't need to try the cast below this.